ios-trivia-game/ViewControllers/MainTabViewController.swift

```diff
@@ -14,6 +14,7 @@ class MainTabViewController: UITabBarController {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        self.navigationController?.isNavigationBarHidden = true
```

Comment:
I ended up hiding the navigationbar completely. let me know if this is an issue for you two